client/components/Profile/ShowComments.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactElement, useEffect } from "react";
+import Paper from "@material-ui/core/Paper";
+
+import Rating from "@material-ui/lab/Rating";
+import Typography from "@material-ui/core/Typography";
+import useApi from "../../hooks/useApi";
+import Loading from "../Routes/Loading";
+import useStyles from "./Profile.styles";
+
+interface Comment {
+  movieName: string;
+  date: number;
+  name: string;
+  body: string;
+  stars: number;
+  _id: number;
+}
+
+interface Props {
+  username: string;
+}
+
+const ShowComment = ({ username }: Props): ReactElement => {
+  const classes = useStyles({});
+  const { resData: data, setUrl } = useApi("", {
+    hotReload: true
+  });
+  useEffect(() => {
+    if (username !== undefined) {
+      setUrl(`/user-comments/${username}`);
+    }
+  }, [username]);
+  if (!data) {
+    return null;
+  }
+  // if (error) {
+  //   return <Error />;
+  // }
+  return (
```

Comment:
Add space !! It's so hard to read without new line

```suggestion
  const { resData: data, setUrl } = useApi("", {
    hotReload: true
  });
  
  useEffect(() => {
    if (username !== undefined) {
      setUrl(`/user-comments/${username}`);
    }
  }, [username]);
  
  if (!data) {
    return null;
  }

  return (
```